src/ProgramBuilder.ts

```diff
@@ -239,16 +238,16 @@ export class ProgramBuilder {
             if (!emitFullPaths) {
                 filePath = path.relative(cwd, filePath);
             }
-
             //load the file text
-            let fileText = await util.getFileContents(pathAbsolute);
-            let lines = util.getLines(fileText);
+            const file = this.program.getFileByPathAbsolute(pathAbsolute);
```

Comment:
This loads the file contents from the in-memory BrsFile/XmlFile. Previously the contents were being loaded from disk, which does not work for in-memory-only files injected by plugins.